Sources/MapboxNavigation/StyleManager.swift

```diff
@@ -12,14 +12,6 @@ public protocol StyleManagerDelegate: AnyObject, UnimplementedLogging {
      */
     func location(for styleManager: StyleManager) -> CLLocation?
     
-    /**
-     Asks the delegate for the view to be used when refreshing appearance. 
-     
-     The default implementation of this method will attempt to cast the delegate to type
-     `UIViewController` and use its `view` property.
-     */
-    func styleManager(_ styleManager: StyleManager, viewForApplying currentStyle: Style?) -> UIView?
```

Comment:
Removing this method would be a backwards compatible change, as noted in https://github.com/mapbox/mapbox-navigation-ios/pull/3764#issuecomment-1065516205. Even though it was optional, technically, application code could’ve previously called this method on an arbitrary class conforming to this protocol; now that code would trigger a compiler error.

Instead of removing this method, just remove the internal calls to it and mark it as deprecated. Also remove the blurb from the changelog about this method being removed.

Reply:
Thanks for the suggestion. This function has been marked as deprecated. The changeLog entry is also updated to the deprecated instead of removed.